central/reports/common/query_builder_test.go

```diff
@@ -70,6 +70,6 @@ func TestBuildQuery(t *testing.T) {
 	rq, err := qb.BuildQuery()
 	assert.NoError(t, err)
 
-	assert.ElementsMatch(t, []string{"Cluster:remote+Namespace:ns1", "Cluster:secured+Namespace:ns2"}, rq.ScopeQueries)
+	assert.ElementsMatch(t, []string{`Cluster:"remote"+Namespace:"ns1"", "Cluster:"secured"+Namespace:"ns2""`}, rq.ScopeQueries)
```

Comment:
It seems this altered query fails in non-postgres unit tests.
There were two strings in the array before the change, and only one after.